packages/edit-post/src/store/constants.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * The identifier for the data store.
+ * @type {string}
+ */
+export const STORE_KEY = 'core/edit-post';
+
+/**
+ * CSS selector string for the admin bar view post link anchor tag.
+ * @type {string}
+ */
+export const VIEW_AS_LINK_SELECTOR = '#wp-admin-bar-view a';
```

Comment:
Aside: It wasn't introduced here, but "View" shows for draft posts with a different ID `wp-admin-bar-preview` (`view` vs. `preview`) and thus wouldn't be updated.

Reply:
> Aside: It wasn't introduced here, but "View" shows for draft posts with a different ID `wp-admin-bar-preview` (`view` vs. `preview`) and thus wouldn't be updated.

Follow-up issue: #15402